core/src/main/java/org/apache/accumulo/core/security/crypto/impl/AESCryptoService.java

```diff
@@ -67,6 +66,7 @@
   private String encryptingKeyManager = null;
   // Lets just load keks for reading once
   private static HashMap<String,Key> decryptingKeys = new HashMap<>();
+  private static SecureRandom sr = CryptoUtils.getSha1SecureRandom();
```

Comment:
You could make this created in init() so it won't have to be static.  We are making a push to try and eliminate static state wherever we can.